tools/update_repo.sh

```diff
@@ -579,6 +585,10 @@ EOF
         initiate_deb_metadata dists/$loop_dist/$component/source/Sources
         for dsc in $ws/$debdir/$loop_dist/$component/*/*/*.dsc ; do
             [ -f $dsc ] || continue
+            if [[ "$arch" != "x86_64" && \
+                  "$deb" != "_${arch}.deb" ]]; then
+              continue
+            fi
```

Comment:
The logic here is also not obvious for me.

Reply:
Now it looks so:

```sh
# Publish source files only on amd64 to avoid file
# name collisions: sources are the same anyway.
[ "${arch}" = "amd64" ] || continue
```